packages/math-input/src/types.ts

```diff
@@ -10,6 +10,12 @@ import {
 import type {CursorContext} from "./components/input/cursor-contexts";
 
 export interface MathQuillInterface {
+    L: "L";
+    R: "R";
+    MathField: (mount: HTMLDivElement, options: any) => MathFieldInterface;
+}
+
+export interface MathFieldInterface {
```

Comment:
Wondering if all this code merits its own `mathquill-types` file.

Reply:
I like the idea of that.